### PR TITLE
Added IoT reference book.

### PR DESCRIPTION
**Pull request by aayush-fadia**

### CODE REVIEW

**Comment by kamatajinkya** [comment]
Add a reviewer

File: iot/README.md [hunk 1]
```diff
@@ -9,4 +9,10 @@ IoT deals with all manner of conventional devices made smart and connected, so t
 
 This requires knowledge of embedded linux (on Raspberry Pi, and other SBCs), Python (The de-facto language for an RPi), microcontrollers, network Connectivity (Firebase, LAN systems), and App Development. The course mentioned is the Android Development Nanodegree, which will give you a fair base to start developing for android. Apart from this, use linux as your daily driver OS, do as much stuff as you can from a terminal (You will mostly SSH into an RPi, no GUI), and you should be fine.Since IoT projects involve wrangling with a whole host of APIs, SDKs and other stuff, there are no courses that encompass it fully, but an app is almost always part of the loop.
 
+## Books
+
+[IoT For Architects](https://drive.google.com/file/d/1QdTIp8jX1EZWd8g0NN01IscJfGnyw2K-/view?usp=sharing)
```
Comment (kamatajinkya) [inline iot/README.md:14]: Do you really want to add a link to a pirated book on an open platform?

**Review by kamatajinkya — CHANGES_REQUESTED** [review]
Not sure if we should add the drive link

File: iot/README.md (hunk 1 above)
Comment (take2rohit) [reply iot/README.md:14]: we should remove that link and rather add only the book name and amazon link only

**Comment by aayush-fadia** [comment]
#52

**Pull request closed without merging**